From f71389ae8f8de243f1e5847cdf5f41524f268694 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Fri, 17 Jun 2011 09:47:06 +0800
Subject: [PATCH] ENGR00151697 MX51 BBG:Correct backlight iomux

Blank DI0 framebuffer will switch off the backlight
of WVGA LCD panel on DI1.
This patch corrects the iomux of backlight controlling
pin, which is configured to a gpio pin.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 arch/arm/mach-mx5/mx51_babbage.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx5/mx51_babbage.c b/arch/arm/mach-mx5/mx51_babbage.c
index daec48e..5819661 100644
--- a/arch/arm/mach-mx5/mx51_babbage.c
+++ b/arch/arm/mach-mx5/mx51_babbage.c
@@ -151,6 +151,7 @@ static iomux_v3_cfg_t mx51babbage_pads[] = {
 
 	MX51_PAD_DI1_PIN12__GPIO3_1,
 	MX51_PAD_DI1_D0_CS__GPIO3_3,
+	MX51_PAD_DI1_D1_CS__GPIO3_4,
 	MX51_PAD_DISPB2_SER_DIN__GPIO3_5,
 	MX51_PAD_DISPB2_SER_DIO__GPIO3_6,
 	MX51_PAD_NANDF_CS0__GPIO3_16,
-- 
1.5.4.4

